getting-started/setup-multi-node-basic.md

```diff
@@ -89,6 +89,38 @@ GRANT USAGE ON FOREIGN SERVER <data node name>, <data node name>, ... TO testrol
 >case all user roles (including future ones) will be able to use the
 >specified data nodes.
 
+## Maintenance tasks [](multi-node-maintenance)
+
+It is highly recommended that the access node is configured to run a
+maintenance job that regularly "heals" any non-completed distributed
+transactions. A distributed transaction ensures atomic execution
+across multiple data nodes and can remain in a non-completed state in
+case a data node reboots or experiences temporary issues. The access
+node keeps a log of distributed transactions so that nodes that
+haven't yet completed their part of the distributed transaction can
+later complete it at the access node's request. The log requires
+regular cleanup to "garbage collect" transactions that have completed
+and heal those that haven't. The maintenance job can be run as a
+user-defined action (custom job):
+
+
+```sql
+CREATE OR REPLACE PROCEDURE data_node_maintenance(job_id int, config jsonb)
+LANGUAGE SQL AS
+$$
+    SELECT _timescaledb_internal.remote_txn_heal_data_node(fs.oid)
+    FROM pg_foreign_server fs, pg_foreign_data_wrapper fdw
+    WHERE fs.srvfdw = fdw.oid
+    AND fdw.fdwname = 'timescaledb_fdw';
+$$;
+
+SELECT add_job('data_node_maintenance', '5m');
+```
+
```

Comment:
Maybe add something like we plan to build this in automatically in the future (if true)?

Reply:
I thought about it, but refrained from doing so for two reasons

1. It isn't yet clear to me how we should do this automatically given that an instance becomes a distributed database when you add the first data node. Then this will add yet another background job for each database, which is subject to background worker settings, etc.
2. I am not sure it is a good idea to make promises about future functionality in the documentation. Such promises are easily forgotten and outdated. I think it is better to collect such future functionality in one place, e.g., roadmap doc or limitations section, where it is easier to maintain.